table/tables/memory_tables.go

```diff
@@ -186,7 +186,7 @@ func (t *MemoryTable) UpdateRecord(ctx context.Context, h int64, oldData, newDat
 }
 
 // AddRecord implements table.Table AddRecord interface.
-func (t *MemoryTable) AddRecord(ctx context.Context, r []types.Datum) (recordID int64, err error) {
+func (t *MemoryTable) AddRecord(ctx context.Context, r []types.Datum, recordDupChecked bool) (recordID int64, err error) {
```

Comment:
How about just use `skipCheck`, the name we are familiar with?

Reply:
we already have skipCheck variable. it may be confused.

Reply:
see line tables.go:419

Reply:
I know, but I think to use another name is more confusing.
It's the same thing in different scopes, like global variable and session variable.